src/components/hiddenInput.js

```diff
@@ -0,0 +1,53 @@
+(() => ({
+  name: 'HiddenInput',
+  icon: 'HiddenInputIcon',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { getActionInput, useText } = B;
+    const isDev = B.env === 'dev';
+    const { actionInputId } = options;
+    const actionInput = getActionInput(actionInputId);
+
+    return (
+      <div className={[classes.root, isDev ? classes.bordered : ''].join(' ')}>
```

Comment:
Why always render the div in runtime only the input element is needed right

Reply:
> Why always render the div in runtime only the input element is needed right

Fixed